src/print/index.ts

```diff
@@ -229,18 +229,21 @@ export function print(path: FastPath, options: ParserOptions, print: PrintFn): D
 
             if (isAttributeShorthand && options.svelteAllowShorthand) {
                 return concat([line, '{', node.name, '}']);
-						} else {
-							const def: Doc[] = [line, node.name];
-							if (node.value !== true) {
-									def.push('=');
-									const quotes = !hasLoneMustacheTag || options.svelteStrictMode;
-
-									quotes && def.push('"');
-									def.push(...path.map(childPath => childPath.call(print), 'value'));
-									quotes && def.push('"');
-							}
-							return concat(def);
-						}
+            } else {
+                const def: Doc[] = [line, node.name];
+                if (node.value !== true) {
+                    def.push('=');
+                    const quotes = !hasLoneMustacheTag || options.svelteStrictMode;
+                    const curlies = quotes && isAttributeShorthand && !options.svelteAllowShorthand;
```

Comment:
I'd suggest removing ` && !options.svelteAllowShorthand`. It will always be true here, but even if it somehow weren't you'd still want those curlies.